dev-python/python-slugify/python-slugify-1.2.6.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+DESCRIPTION="A Python Slugify application that handles Unicode"
+HOMEPAGE="https://github.com/un33k/python-slugify https://pypi.org/project/python-slugify/"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="dev-python/unidecode[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}"
+
+distutils_enable_tests nose
```

Comment:
This doesn't seem to be doing anything. You should test it yourself before committing..
```
>>> Test phase: dev-python/python-slugify-1.2.6
 * python3_6: running distutils-r1_run_phase python_test

----------------------------------------------------------------------
Ran 0 tests in 0.007s

OK
 * python3_7: running distutils-r1_run_phase python_test

----------------------------------------------------------------------
Ran 0 tests in 0.009s
```

Reply:
I did test this, but I somehow oversaw that this specific version comes without tests.

So I removed the tests from the ebuild, sorry for the confusion.

Reply:
I mean, the Github release does include tests.
https://github.com/un33k/python-slugify/tree/beca50eaff20ca20652df782354a8f184c1eaf4c

Please us that instead. Looks like `${EPYTHON} test.py || die "tests failed with ${EPYTHON}"` inside `python_test()` is enough to get them to work.

Reply:
Yes, but the release `tar.gz` doesn't contain the tests anymore.

Reply:
See the file linked here: https://pypi.org/project/python-slugify/1.2.6/#files

Reply:
Use https://github.com/un33k/python-slugify/archive/1.2.6.tar.gz

Reply:
Done. Now the tests also work with `distutils_enable_tests` :)